src/main/java/org/testobject/espressorunner/EspressoRunner.java

```diff
@@ -4,9 +4,9 @@
 import com.google.gson.GsonBuilder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.testobject.api.TestObjectClient;
+import org.testobject.api.v2.TestObjectClientV2;
```

Comment:
not the PR for this but I don't think it's useful to have `V2` in the class name when it's already in the module path

Reply:
I don't like that "V2" name too. Right now Java-Api is already published. Java-Api should be cleaned out, it contains old things. After that V2 can be renamed.